public/img/github.svg

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no"?>
 <svg width="19px" height="18px" viewBox="0 0 19 18" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
   <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
-    <g transform="translate(-54.000000, -66.000000)" fill="#95AABC">
+    <g transform="translate(-54.000000, -66.000000)" fill="#ff5e50">
```

Comment:
i agree in general, the problem is that this is a img element (w/ SVG src), so i'm not the class would work here